tests/rule_based_profiler/data_assistant/test_volume_data_assistant.py

```diff
@@ -370,22 +373,22 @@ def test_volume_data_assistant_plot_descriptive(
         "data_asset_name": "my_reports",
     }
 
-    data_assistant: DataAssistant = VolumeDataAssistant(
+    data_assistant: VolumeDataAssistant = VolumeDataAssistant(
```

Comment:
@NathanFarmer To me this is also an anti-pattern.  Our `DataAssistant` classes implement common interfaces.  Hence, using polymorphism, we should be able to call all interface methods through the base `DataAssistant` class.

Reply:
✅ Taken care of offline